1_turb_abl_fsi/template_files/nrel5mw_amr.inp

```diff
@@ -0,0 +1,199 @@
+#.......................................#
+#            SIMULATION STOP            #
+#.......................................#
+time.stop_time               = -1     # Max (simulated) time to evolve
+time.max_step                = 31440 #-1          # Max number of time steps
+#.......................................#
+#         TIME STEP COMPUTATION         #
+#.......................................#
+time.fixed_dt         =   0.003443526170799   # Use this constant dt if > 0
+time.cfl              =   0.95       # CFL factor
+#.......................................#
+#            INPUT AND OUTPUT           #
+#.......................................#
+io.restart_file = {AMR_RESTART_FILE}
+time.plot_interval            =  500       # Steps between plot files
+time.checkpoint_interval      =  500       # Steps between checkpoint files
+time.plot_start               =  30000
+#.......................................#
+#               PHYSICS                 #
+#.......................................#
+incflo.gravity        =  0.0  0.0 -9.81  # Gravitational force (3D)
+incflo.density        =  1.0             # Reference density
+incflo.use_godunov = 1
+incflo.godunov_type = "weno_z"
+incflo.velocity = 9.8726896031426 5.7 0.0
+
+transport.viscosity = 0.0
+transport.laminar_prandtl = 0.7
+transport.turbulent_prandtl = 0.3333
+
+turbulence.model = OneEqKsgsM84
+TKE.source_terms = KsgsM84Src
+
+incflo.physics = ABL
+ICNS.source_terms = BoussinesqBuoyancy CoriolisForcing BodyForce ABLMeanBoussinesq
+
+BoussinesqBuoyancy.reference_temperature = 300.0
+BoussinesqBuoyancy.read_temperature_profile = true
+BoussinesqBuoyancy.tprofile_filename = {AMR_TPROFILE_FILE}
+
+CoriolisForcing.east_vector = 1.0 0.0 0.0
+CoriolisForcing.north_vector = 0.0 1.0 0.0
+CoriolisForcing.latitude = 40.0
+CoriolisForcing.turn_off_vertical_force = True
+CoriolisForcing.rotational_time_period = 86400.0
+
+BodyForce.magnitude = -0.000349710865169856 0.0011134941091590048 0.0
```

Comment:
Could this be added to the scripting, in case the precursor changes?

Reply:
I think this request is for a different iteration.  I don't think we have any plans to change the precursor.